src/test/java/org/minions/devfund/abel/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,123 @@
+package org.minions.devfund.abel.moviedatabase;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class for Movie Database Unit Tests.
+ */
+public class MovieDatabaseTest {
+
+    /**
+     * Test AddMovie.
+     */
+    @Test
+    public void testAddMovieNewMovieAndActors() {
+        MovieDatabase movieDatabase = new MovieDatabase();
```

Comment:
You are repeating this code in all your tests
Please add it to a @Before method.